lblod/frontend-lokaal-mandatenbeheer
# Ben/lmb 422 bestuursperiode concept

## Description

Add new bestuursperiode model, update current bestuursperiode selector and refactor the uses of the bestuursperiode util and tijdsspecialisaties service to take advantage of these new concepts.

## How to test

The bestuursperiode selector is used in three different places, all in the organen route (index, orgaan and fracties), be sure to check these. But these changes could have a big impact, so it is a good idea to also just click around a bit and iteract with different bestuursorganen in de tijd etc.

## Links to other PR's

- https://github.com/lblod/app-lokaal-mandatenbeheer/pull/135
